File: ago-dic-2020/Luis Enrique Cázares Martínez/Practica1/Practica1.py

numeros = input("Dame dos numeros separados por espacio pliz: ")
listanumeros = numeros.split(" ")
x = int(listanumeros[0])
y = int(listanumeros[1])

if (x >= -100 and y <= 100):
    print(f"Los parametros de 'x' y 'y' son correctos\nLa suma de ambos es {x + y}")

File: ago-dic-2024/Carlos_Silva/Practica1.java
package Silva_Carlos;

import java.util.Scanner;

public class Practica1 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        //INTRODUCIR LOS NUMEROS ENTEROS SEPARADOS POR UN ESPACIO (INPUT)

        System.out.println("Introduce dos numeros enteros: ");
        int a = sc.nextInt();
        int b = sc.nextInt();

        int suma = a + b;

        //SALIDA DE LOS NUMEROS ENTEROS SUMADOS (OUTPUT)

        System.out.println("Tu resultado es: " + suma);
    }
}
